src/components/group-management-menu/index.tsx

```diff
@@ -81,7 +81,7 @@ export class GroupManagementMenu extends React.Component<Properties, State> {
         side='bottom'
         alignMenu='end'
         onOpenChange={this.handleOpenChange}
-        trigger={<IconDotsHorizontal size={24} isFilled />}
+        trigger={<IconButton Icon={IconDotsHorizontal} size={32} isFilled onClick={() => {}} />}
```

Comment:
`onClick` doesn't need to do anything here as `trigger` handles the click functionality.

Reply:
Alternatively we could add a wrapper around `<IconDotsHorizontal size={24} isFilled />` and set the hover styles with another className, but adding `IconButton` instead won't require us to add any other styles.